Src/FluentAssertions/Formatting/DateTimeOffsetValueFormatter.cs

```diff
@@ -21,6 +21,7 @@ public bool CanHandle(object value)
         public void Format(object value, FormattedObjectGraph formattedGraph, FormattingContext context, FormatChild formatChild)
         {
             DateTimeOffset dateTimeOffset;
+            bool significantOffset = false;
```

Comment:
Can you try to move those changes in a separate commit so we can easily see why this change was needed?

Reply:
The formatter changes? I'll do anything you ask for, as soon as I understand what it is :)

My rationale for the formatting change was:

1. Without the offset mentioned, even if it's TimeSpan.Zero, the assertion messages can become cryptic. "Exptected 2021-06-01 +1h, but it was 2021-06-01"
2. I'm just guessing, but I think it's actually more common to deal with DateTimeOffsets which have a non-zero offset, so the change shouldn't be that big of a deal.
3. The offset is skipped for plain DateTime. I thought that would make sense.

Reply:
> My rationale for the formatting change was:

@jnyrup what do you think? Makes sense to me.

> The formatter changes? I'll do anything you ask for, as soon as I understand what it is :)

To split your changes into two commits. One with the formatter changes (with the rational in the title and description), and one with the new `DateTimeOffset` assertions. See also https://www.continuousimprover.com/2020/03/keep-source-control-history-clean.html

Reply:
Also, using "UTC" as the offset was just a choice that seemed OK at the time. UTC is not an offset, it's a time zone. There are other time zones which have a zero offset, so something like "Z", "+0h" or even "±0h" could make more sense.

Reply:
I think `+0h` would be clearer.

Reply:
Agreed. Done. I will fix the commits according to you liking if and when we conclude the matter. The documentation will need some additions too.